### PR TITLE
Issue #67 [FEATURE] Balance resources section

### PR DESCRIPTION
**Pull request by pattimcletchie**
## Related [Issue 67](https://github.com/web3community/web3community.github.io/issues/67)

- Request to make the footer more balanced

Closes: #67

### Describe the changes you've made
I removed `flex-shrink-0` and added `flex-1` to the column containing "Socials" and the social icons. This helps even out all of the columns bringing them closer together.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I have tested this change locally.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 
135774018-7b32db54-15c9-446d-b53a-bef8def855fd
 | 
Screen Shot 2021-10-04 at 8 37 59 PM
 |
 
## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "web3community.github.io",
```
Comment (Panquesito7) [inline package-lock.json:8]: Please add this line back.

File: src/_includes/partials/footer.html [hunk 2]
```diff
@@ -91,7 +92,8 @@
             2021. Made with ❤️ by
             <a href="{{settings.url}}" target="_blank" class="underline">{{settings.author}}</a>
             •
-            <a href="https://github.com/web3community/web3community.github.io" class="underline" target="_blank" rel="noopener">View on Github</a>
+            <a href="https://github.com/web3community/web3community.github.io" class="underline" target="_blank"
+                rel="noopener">View on Github</a>
         </small>
     </div>
-</footer>
+</footer>
```
Comment (Panquesito7) [inline src/_includes/partials/footer.html:99]: Missing an endline.
```suggestion
</footer>

```

File: src/_includes/partials/footer.html [hunk 3]
```diff
@@ -2,36 +2,36 @@
...
-            </div>
-            <!--socials icons ended-->
+                    <a aria-hidden='true'
+                        class='text-white fa fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://github.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
+                </div>
+                <!--socials icons ended-->
 
-            <!--menu start-->
+                <!--menu start-->
```
Comment (Panquesito7) [inline src/_includes/partials/footer.html:34]: I think the indentation was fine. Please change it back as to how it was (same in other lines above).

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by vinzvinci — COMMENTED** [review]
Thanks for the review @Panquesito7, please follow @Panquesito7 request

File: src/_includes/partials/footer.html (hunk 3 above)
Comment (pattimcletchie) [reply src/_includes/partials/footer.html:34]: yeah, I think I have Prettier configed to run on save for all projects on my workspace... sorry about that.

**Comment by pattimcletchie** [comment]
@Panquesito7 it's ready for re-review.

**Comment by vinzvinci** [comment]
Hi, have you participated in hacktoberfest? because we labeled this for hacktoberfest, so your contributions will count there :) https://hacktoberfest.digitalocean.com/

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by shuklaritvik06 — APPROVED** [review]
LGTM🎉

**Review by Panquesito7 — APPROVED** [review]
Awesome work! Thanks! 🚀

**Comment by mkubdev** [comment]
Nice one again, thanks 🎃

**Pull request merged**